python/tchannel/thrift/scheme.py

```diff
@@ -28,7 +28,7 @@
 from .. import scheme
 
 
-class ThriftArgScheme(scheme.ArgScheme):
+class ThriftArgScheme(scheme.RawArgScheme):
```

Comment:
@junchaowu what object benefits from this inheritance structure? You aren't calling into the parent, and `ThriftArgScheme` re-implements all the functionality of the class its extending. This feels a bit more like an interface, which I'm fine with.